contrib/reformat-code.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2017 Dell Inc.
+#
+# SPDX-License-Identifier: LGPL-2.1+
+#
+
+import argparse
+import os
+import sys
+import subprocess
+
+CLANG_FORMATTERS = [
+    "clang-format-13",
+    "clang-format",
```

Comment:
I think we explicitly only want 13+ right?

Reply:
We do. I believe normally the second one should not exist. At our Google environment we do have another installation coming from our repos and installed at `/usr/bin/clang-format`.